src/App.tsx

```diff
@@ -1,39 +1,66 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
 function getRandomName(): string {
   const value = Date.now().toString().slice(-4);
 
   return `Clock-${value}`;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+type Props = {};
 
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
+interface State {
+  hasClock: boolean;
+  clockName: string;
+}
+
+export class App extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    hasClock: true,
+    clockName: 'Clock-0',
+  };
+
+  intervalId: number | undefined;
```

Comment:
You can give this a default value of 0 and keep it always as a number
```suggestion
  intervalId = 0;
```